tests/test_packages/test_connections/test_p2p_libp2p/test_fault_tolerance.py

```diff
@@ -42,7 +43,7 @@
 )
 
 
-DEFAULT_PORT = 10234
+ports = itertools.count(start=10234)
```

Comment:
Why not use one global var set in conftest? Otherwise there can be conflicts across files...depending on how pytest is configured this can be an issue.

Reply:
I like that a lot. I was considering a helper function in the libp2p test module, but I like your suggestion better